client/src/pages/Results/GeneResult.tsx

```diff
@@ -0,0 +1,368 @@
+    ...selectedVariants.map((v) => ({ type: 'variant', value: v })),
+    ...selectedDiseases.map((d) => ({ type: 'disease', value: d })),
+    ...selectedTherapies.map((t) => ({ type: 'therapy', value: t })),
+    ...selectedEvidenceLevels.map((e) => ({ type: 'evidence_level', value: e })),
+    ...selectedSignificance.map((s) => ({ type: 'significance', value: s })),
+    ...selectedSources.map((src) => ({ type: 'source', value: src })),
+  ]
+
+  const removeFilter = (filter: { type: string; value: string }) => {
+    switch (filter.type) {
+      case 'variant':
+        setSelectedVariants((prev) => prev.filter((v) => v !== filter.value))
+        break
+      case 'disease':
+        setSelectedDiseases((prev) => prev.filter((d) => d !== filter.value))
+        break
+      case 'therapy':
+        setSelectedTherapies((prev) => prev.filter((t) => t !== filter.value))
+        break
+      case 'evidence_level':
+        setSelectedEvidenceLevels((prev) => prev.filter((e) => e !== filter.value))
+        break
+      case 'significance':
+        setSelectedSignificance((prev) => prev.filter((s) => s !== filter.value))
+        break
+      case 'source':
+        setSelectedSources((prev) => prev.filter((s) => s !== filter.value))
+        break
+    }
+  }
+
+  return (
+    <>
+      <Header />
+      <Box id="result-page-container" m={5}>
+        {loading && <CircularProgress />}
+        {error && <Alert severity="error">{error}</Alert>}
+        {!loading && (
+          <Box>
+            <Box
+              id="results-info-container"
+              sx={{ backgroundColor: 'white', padding: 5, borderRadius: 2 }}
+            >
+              <Typography variant="h4" mb={2} fontWeight="bold">
+                {searchQuery}
+              </Typography>
+              <Typography variant="h6" mb={2} fontWeight="bold" color="darkgrey">
+                Aliases: {aliases.join(', ')}
+              </Typography>
+              <Typography variant="body1" mb={2}>
+                {description}
+              </Typography>
+            </Box>
+            <Box
+              id="results-table-container"
+              sx={{ backgroundColor: 'white', padding: 5, borderRadius: 2, marginTop: 2 }}
+            >
+              <Tabs
+                onChange={(_, value) => setActiveTab(value)}
+                value={activeTab}
+                sx={{ marginBottom: 2 }}
+              >
+                <Tab
+                  label={`Therapeutic (${filteredByTab.therapeutic.length})`}
+                  value="therapeutic"
+                />
+                <Tab label={`Diagnostic (${filteredByTab.diagnostic.length})`} value="diagnostic" />
+                <Tab label={`Prognostic (${filteredByTab.prognostic.length})`} value="prognostic" />
+              </Tabs>
+              <Typography variant="h6" mb={2} fontWeight="bold">
+                {activeTab} Search Results ({filteredResults?.length})
+              </Typography>
```

Comment:
I think this should be in a div with 16px of padding just like the stuff above and below it

<img width="495" height="229" alt="Screenshot 2025-10-06 at 4 12 35 PM" src="https://github.com/user-attachments/assets/7f2c12ea-9404-4a94-b58c-8accd4735888" />